NitroxServer/Communication/Packets/Processors/PlayerJoiningMultiplayerSessionProcessor.cs

```diff
@@ -39,7 +39,7 @@ public override void Process(PlayerJoiningMultiplayerSession packet, NitroxConne
 
             List<EquippedItemData> equippedItems = player.GetEquipment();
             List<NitroxTechType> techTypes = equippedItems.Select(equippedItem => equippedItem.TechType).ToList();
-            List<ItemData> inventoryItems = GetInventoryItems(player.GameObjectId);
+            List<ItemData> inventoryItems = GetInventoryItems(player.GameObjectId);            
```

Comment:
Very minor issue: For some reason seems to be an unneeded white space increase here

Reply:
Removed